providers/softlayer/src/test/java/org/jclouds/softlayer/compute/SoftLayerComputeServiceContextLiveTest.java

```diff
@@ -61,13 +62,15 @@ public void testLaunchClusterWithMinDisk() throws RunNodesException {
                       new SshjSshClientModule()))
               .build(ComputeServiceContext.class);
 
+      for (ComputeMetadata computeMetadata : context.getComputeService().listNodes()) {
+         System.out.println(computeMetadata);
```

Comment:
Use `getAnonymousLogger()` instead if you want to print feedback when running tests.